docs/Browser Support.md

```diff
@@ -0,0 +1,73 @@
+# Browser Support
+
+<!-- @import "[TOC]" {cmd="toc" depthFrom=2 depthTo=6 orderedList=false} -->
+
+<!-- code_chunk_output -->
+
+- [Guidelines](#guidelines)
+- [Polyfills](#polyfills)
+
+<!-- /code_chunk_output -->
+
+VTEX browser support is split in two contexts: `storefront` and `admin areas`.
+
+For `storefronts`, we strive to support all major browsers all the way to IE 11:
```

Comment:
Maybe we should add the we guarantee that the store **works**. The layout may vary depending on the browser due to CSS browser support.

Reply:
Done!